src/core/operation_client.rs

```diff
@@ -107,34 +112,26 @@ impl OperationHandler {
 impl Default for OperationHandler {
     fn default() -> Self {
         OperationHandler {
-            converter: Box::from(ProtobufConverter {}),
-            wire_protocol_version_maj: 1,
-            wire_protocol_version_min: 0,
-            content_type: BodyType::Protobuf,
-            accept_type: BodyType::Protobuf,
+            content_type: Box::from(ProtobufConverter {}),
```

Comment:
Maybe the field name `content_converter` is more appropriate?

Reply:
Yeah, I guess, especially since it's public